scripts/encapp_tool/adb_cmds.py

```diff
@@ -20,9 +20,9 @@ def run_cmd(cmd: str, debug: int = 0) -> Tuple[bool, str, str]:
     try:
         if debug > 0:
             print(cmd, sep=" ")
-        process = Popen(cmd, shell=True, stdout=PIPE, stderr=PIPE)  # noqa: P204
-        stdout, stderr = process.communicate()
-        ret = True if process.returncode == 0 else False
+        with Popen(cmd, shell=True, stdout=PIPE, stderr=PIPE) as process:
```

Comment:
nice. 

Does this mean that we're leaking if we don't do close the process?

Reply:
I may if there is failure, https://bugs.python.org/issue43308.